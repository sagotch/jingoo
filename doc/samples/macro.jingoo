{% macro li (content) %}<li>{{ content }}</li>{% endmacro -%}
<ol>
  {{ li ('Item1') }}
  {{ li ('Item2') }}
  {{ li ('Item3') }}
</ol>
