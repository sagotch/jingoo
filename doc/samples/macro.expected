<ol>
  <li>Item1</li>
  <li>Item2</li>
  <li>Item3</li>
</ol>
